invenio_cli/commands/local.py

```diff
@@ -101,62 +143,72 @@ def update_statics_and_assets(self, force, flask_env="production", log_file=None
             "install": "Installing JS dependencies...",
         }
 
-        with env(FLASK_ENV=flask_env):
+        with env(FLASK_DEBUG="true"):
             for op in ops:
                 if callable(op):
                     response = op()
                 else:
                     if op[-1] in messages:
                         click.secho(messages[op[-1]], fg="green")
                     response = run_interactive(
-                        op, env={"PIPENV_VERBOSITY": "-1"}, log_file=log_file
+                        op,
+                        env={"PIPENV_VERBOSITY": "-1"},
+                        log_file=log_file,
                     )
                 if response.status_code != 0:
                     break
         return response
 
-    def run(self, host, port, debug=True, services=True, celery_log_file=None):
-        """Run development server and celery queue."""
+    def lock(self):
+        """Lock javascript dependencies."""
+        from flask_collect import Collect
```

Comment:
this requires access to the `my-site` venv and won't work with e.g. a global install of `invenio-cli`.

this seems to be only used in a _new_ `invenio-cli assets lock` command which is used as part of the "reusable frontend lock files" feature.
i would suggest splitting that new command out of this PR, into a new one.

that would basically still enable the "reusable frontend lock file" mechanism, even if less convenient (at least until the improvement PR comes along).
to phrase it more positively, it would be available now as a sort of "expert feature".